examples/tutorials/Cell_Counting_Tutorial.ipynb

```diff
@@ -0,0 +1,503 @@
+{
```

Comment:
<p>Link to another tutorial that introduces callbacks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Done. Please review